src/docs/software/using/julia.md

```diff
@@ -0,0 +1,295 @@
+    are shared, packages installed on one node will immediately be available on
+    all nodes on the cluster.
+
+#### Installing packages
+
+You can first check the status of packages installed on Julia using the status function of the `Pkg` module:
+```
+julia> Pkg.status()
+No packages installed.
+```
+
+You can then add packages using the add function of the `Pkg` module:
+```
+julia> Pkg.add("Distributions")
+INFO: Cloning cache of Distributions from git://github.com/JuliaStats/Distributions.jl.git
+INFO: Cloning cache of NumericExtensions from git://github.com/lindahua/NumericExtensions.jl.git
+INFO: Cloning cache of Stats from git://github.com/JuliaStats/Stats.jl.git
+INFO: Installing Distributions v0.2.7
+INFO: Installing NumericExtensions v0.2.17
+INFO: Installing Stats v0.2.6
+INFO: REQUIRE updated.
+```
+
+Using the status function again, you can see that the package and its dependencies have been installed:
+```
+julia> Pkg.status()
+Required packages:
+ - Distributions                 0.2.7
+Additional packages:
+ - NumericExtensions             0.2.17
+ - Stats                         0.2.6
+```
+#### Updating Packages
+
+The update function of the `Pkg` module can update all packages installed:
+```
+julia> Pkg.update()
+INFO: Updating METADATA...
+INFO: Computing changes...
+INFO: Upgrading Distributions: v0.2.8 => v0.2.10
+INFO: Upgrading Stats: v0.2.7 => v0.2.8
+```
+#### Removing packages
+
+The remove function of the `Pkg` module can remove any packages installed as well:
+```
+julia> Pkg.rm("Distributions")
+INFO: Removing Distributions v0.2.7
+INFO: Removing Stats v0.2.6
+INFO: Removing NumericExtensions v0.2.17
+INFO: REQUIRE updated.
+
+julia> Pkg.status()
+Required packages:
+ - SHA                           0.3.2
+
+julia> Pkg.rm("SHA")
+INFO: Removing SHA v0.3.2
+INFO: REQUIRE updated.
+
+julia> Pkg.status()
+No packages installed.
+```
+Working with packages in Julia is simple!
+
+### Examples
+
+
+
+Julia can natively spawn parallel workers across multiple compute nodes, without using MPI.
+
+There are 2 modes of operation:
+
+1\. ClusterManager; in which you start Julia, and from within Julia, you
+spawn workers which will actually submit jobs to the scheduler,
+executing instructions within those jobs.
+
+2\. --machine file; here you submit a SLURM job and run the Julia
+interpreter in parallel mode within the job's resources.
+
+Mode 2 is easier to use, and more convenient, since you have all your
+resources available and ready to use when the job starts. In mode 1,
+you'll need to wait for jobs to be dispatched and executed inside Julia.
+
+Here is a quick example on how to use the --machinefile mode on
+Sherlock.
+
+Given following Julia script (parallel\_test.jl) that will print a line
+with the process id and the node it's executing on, in parallel:
+
+    @everywhere println(@sprintf("process: %d on host %s", myid(), gethostname()))
+
+You can submit the following job:
+```bash tab="juliaTest.sbatch"
```

Comment:
Also needs separating empty line for proper Markdown parsing, apparently.
```suggestion

```bash tab="juliaTest.sbatch"
```